src/nfts/NftsInfoCarousel.test.tsx

```diff
@@ -9,13 +9,14 @@ import { createMockStore, getMockStackScreenProps } from 'test/utils'
 import { mockNftAllFields, mockNftMinimumFields, mockNftNullMetadata } from 'test/values'
 
 jest.mock('src/utils/Logger')
+jest.mock('react-native-video', () => 'ReactNativeVideo')
 
 describe('NftsInfoCarousel', () => {
   beforeEach(() => {
     jest.clearAllMocks()
   })
 
-  it('renders correctly with one Nft', () => {
+  it('renders correctly with one Video Nft', () => {
```

Comment:
Why are we updating this test just for video?

Reply:
The first test nft is now a video, we could add another test for a single image NFT, but the behavior in the carousel is very similar regardless of image or video.

Reply:
if the fact that it's a video doesn't really matter, maybe we shouldn't change the test names, no?